hub/powertoys/color-picker.md

```diff
@@ -46,45 +46,52 @@ The copied color will be stored in your clipboard in the format that is configur
 
 ## Using the Color Picker editor
 
-The editor lets you see the history of (up to 20) picked colors and copy their representation in any predefined string format. In PowerToys Settings, you can configure which color formats are visible in the editor and in what order that they appear.
+The Color Picker editor stores a history of up to 20 picked colors and lets you copy them to the clipboard. You can choose which color formats are visible in the editor in **Color formats** in PowerToys Settings.
 
-The color in the middle represents your currently selected color from the colors history. By clicking on it, the fine-tuning control will appear, where you can change HSV or RGB values. Using the **Select** button will add the new color into the colors history.
+The colored bar at the top of the Color Picker editors lets you:
 
-The editor allows you to fine-tune any picked color or get a new similar color. Editor suggests two lighter and two darker shades of the currently selected color. Selecting any of those alternative shades will add it to the history of picked colors.
+* fine tune your chosen color
+* pick a similar color
+
+To fine tune your chosen color, select the central color in the color bar. The fine-tuning control lets you change the color's **HSV**, **RGB**, and **HEX** values. **Select** adds the new color to the colors history.
+
+To choose a similar color, select one of the segments on the left and right edges of the color bar. The Color Picker editor suggests two lighter shades on the left of the bar, and two darker shades on the right of the bar. Selecting one of these similar colors adds that color to the colors history.
 
 ![ColorPicker Editor window.](../images/pt-colorpicker-editor.gif)
 
-To remove any color from the colors history, right-click a color and select **Remove**.
+To remove a color from the colors history, right-click a color and select **Remove**.
 
-To export the colors history, right-click a color and select **Export**. The exported values can be grouped by colors or formats.
+To export the colors history, right-click a color and hover over **Export**. You can group the exported values by colors or formats.
```

Comment:
This is another case where touch input isn't accounted for. "Hover over" isn't possible when using touch. I would either leave "select" or do something like "hover over or tap"... Select is probably the simplest option, unless you have another idea.

Reply:
Good point. I'd followed some examples in [the style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/right-click) in my new commit.